FAQ item for OpenCart and WooCommerce generic gateway (#1063)
MultiSafepay/docs: content/integrations/plugins/opencart/faq/generic-gateways.md

---
title : "Generic gateways"
meta_title: "Generic gateways - MultiSafepay Docs"
meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
---

> The OpenCart plugin offer a generic gateway; that can be found in plugin settings page, listed in payment methods tab.

Reviewer: ```suggestion
The OpenCart plugin offers a generic gateway. The generic gateway can be found on the plugin settings page, listed in the payment methods tab.
```

> All of our payment methods are supported. It is possible to set the gateway code, gateway logo, gateway label and whether the shopping cart should be included in refund transactions; which is required if the gateway code used is a billing suite payment method.

Reviewer: ```suggestion
All of our payment methods are supported. It is possible to set the gateway code, gateway logo, gateway label, and whether the shopping cart should be included in refund transactions. Specifying whether the shopping cart should be included in refund transactions is required if the gateway code used is that of a billing suite payment method.
```

> In the same way than others gateways, is possible also filter the generic gateway by geo zone, currency, min amount, max amount, customer groups, and a set a custom initial order status.

Reviewer: ```suggestion
Similar to other gateways, it is possible to filter the generic gateway by:

* geographic zone
* currency
* min amount
* max amount
* customer groups

Furthermore, you can set a custom initial order status.
```

> Full refunding actions and backend orders are fully supported.

Reviewer: ```suggestion
Full refunds and backend orders are fully supported.
```
